examples/vizdoom/env.py

```diff
@@ -105,17 +118,27 @@ def step(self, action):
         elif self.game.is_episode_finished():
             done = True
             info["TimeLimit.truncated"] = True
-        return self.obs_buffer, reward, done, info
+        return self.obs_buffer, reward, done, info.get("TimeLimit.truncated", False), info
 
-    def render(self):
+    def render(self) -> None:
         pass
 
-    def close(self):
+    def close(self) -> None:
         self.game.close()
 
 
-def make_vizdoom_env(task, frame_skip, res, save_lmp, seed, training_num, test_num):
-    test_num = min(os.cpu_count() - 1, test_num)
+def make_vizdoom_env(
+    task: str,
+    frame_skip: int,
+    res: tuple[int],
+    save_lmp: bool = False,
+    seed: int | None = None,
+    training_num: int = 10,
+    test_num: int = 10,
+) -> tuple[Any | Env, Any | ShmemVectorEnv, Any | ShmemVectorEnv]:
```

Comment:
I think `Any | ...` doesn't make much sense. Why do we need `Any`? If there's no better type, maybe just leave `Any` for now? 

Does this maybe merit a separate issue?

Reply:
Thx for spotting it. It should just be `tuple[Env, ShmemVectorEnv, ShmemVectorEnv]`.
I must have used VSCode's double-click to insert the type annotation there.